langlib/langlib-test/src/test/resources/test-src/arraylib_test.bal

```diff
@@ -231,13 +232,6 @@ function testReverseRecord() {
         {name: "John Doe", age: 25, designation: "Software Engineer"}]);
 }
 
-function testArrayReverseEquality() {
-    int[] x = [1, 2, 3, 4, 5];
-    int[] y = x.reverse();
-    assertValueEquality(x == y, true);
```

Comment:
I think we are still modifying the argument. Here `assertValueEquality(x == y, true)`
Result should be false but it is still true. Instead of removing these we can keep them and revert the assertions.